app/src/main/kotlin/at/bitfire/davdroid/ui/AccountsActivity.kt

```diff
@@ -6,53 +6,17 @@ package at.bitfire.davdroid.ui
 
 import android.content.Intent
 import android.os.Bundle
-import androidx.activity.compose.setContent
 import androidx.appcompat.app.AppCompatActivity
-import at.bitfire.davdroid.ui.account.AccountActivity
-import at.bitfire.davdroid.ui.intro.IntroActivity
-import at.bitfire.davdroid.ui.setup.LoginActivity
-import dagger.hilt.android.AndroidEntryPoint
-import javax.inject.Inject
 
-
-@AndroidEntryPoint
+@Deprecated("Automatically redirects to MainActivity. Should be removed in the future.")
```

Comment:
Good to mark it somehow, but we don't have specific plans to remove it yet. May it will be there for years until all the other apps have changed the reference.

So I'd prefer to just add a comment why this is still needed, without `@Deprecated`.